src/common/Toast/Toast.tsx

```diff
@@ -0,0 +1,81 @@
+import ReactDOM from 'react-dom';
+import React, { useEffect, useState } from 'react';
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert, { AlertProps, AlertColor } from '@mui/material/Alert';
+
+import { useAppDispatch } from '../../configureStore';
+// import { resetError } from '../../../store/auth-slice';
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  props,
+  ref
+) {
+  // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
add the following to .eslintrc.json "rules" section :

"react/jsx-props-no-spreading": "off"

Reply:
added successfully